pkg/services/dns/dns_config_unix.go

```diff
@@ -11,6 +12,11 @@ func getDNSHostAndPort() (string, string, error) {
 	if err != nil {
 		return "", "", err
 	}
+
+	if len(conf.Servers) == 0 {
+		return "", "", errors.New("No DNS servers configured in /etc/resolv.conf")
+	}
+
```

Comment:
I'd favour something like 
```
var errEmptyResolvConf = errors.New("no DNS servers configured in /etc/resolv.conf") // lower case at the beginning 

func getDNSHostAndPort() (string, string, error) {
    ...
    	if len(conf.Servers) == 0 {
		return "", "", errEmptyResolvConf
	}
```
and then instead of 
```
if err != nil {
	ginkgo.Skip("Skipping test, failed to setup DNS server")
}
```
use the more specific `if errors.Is(err, errEmptyResolvConf)` so that we don't ignore other errors

Reply:
I've implemented your suggestion. Thanks!